backend/src/app.controller.ts

```diff
@@ -1,8 +1,10 @@
 import { Controller, Get } from '@nestjs/common';
 
+// eslint-disable-next-line import/extensions
```

Comment:
I've noticed that this rule is repeated quite often. How about disabling it in the config?

Reply:
This rule disabling for now. I figure out later this for not disabling